versions/3.0.md

```diff
@@ -786,9 +734,89 @@ schema:
   type: file
 ```
 
+#### <a name="requestBodyObject"></a>Request Body Object
+
+Describes a single request body.
+
+If the `requestBody` attribute is not present, or `schema` is not defined, the request body is not required.
+
+The serialization of the request body should be specified by the `consumes` attribute.
```

Comment:
I'll try to rephrase a bit.

I'd like to spend more time tackling this concept in another PR, where I'm planning to move the media type up one level, something like:
```
request:
  application/json:
    schema: 
      $ref: "#/definitions/User"
```

I think once it's in this format, it's more self-descriptive.